File: 01_Basico/04 - Funções_Modulos/Resposta_em_Python/questao05.py
def converter_para_farenheit(valor_celsius):
    saida_farenheit = valor_celsius * 1.8 + 32
    return saida_farenheit

def converter_para_celsius(valor_farenheit):
    saida_celsius = (valor_farenheit - 32) / 1.8
    return saida_celsius

def opcao_media(tipo_media):
    if tipo_media[0].upper() == "C":
        return f"O valor da medida convertido de °F para °C é {converter_para_celsius(valor_temperatura)}" 
    
    return f"O valor da medida convertido de °C para °F é {converter_para_farenheit(valor_temperatura)}"
    
    
    
medida_escolhida = input("A unidade de medida está em Celsius(°C) ou Farenheit(°F):")
valor_temperatura = float(input("Digite o valor da temperatura: "))

print(opcao_media(medida_escolhida))

File: 01_Basico/04 - Funções_Modulos/Resposta_em_JavaScript/questao05.js
function converter_para_farenheit(valor_celsius) {
    saida_farenheit = valor_celsius * 1.8 + 32
    return saida_farenheit
}

function converter_para_celsius(valor_farenheit) {
    saida_celsius = (valor_farenheit - 32) / 1.8
    return saida_celsius
}

function opcao_media(tipo_media) {
    if (tipo_media[0].toUpperCase() == "C") {
        return `O valor da medida convertido de °F para °C é ${converter_para_celsius(valor_temperatura)}`
    }
    return `O valor da medida convertido de °C para °F é ${converter_para_farenheit(valor_temperatura)}`
}
    
    
    
medida_escolhida = prompt("A unidade de medida está em Celsius(°C) ou Farenheit(°F):")
valor_temperatura = parseFloat(prompt("Digite o valor da temperatura: "))

console.log(opcao_media(medida_escolhida))